x-pack/plugins/security_solution/server/lib/telemetry/receiver.ts

```diff
@@ -443,7 +444,10 @@ export class TelemetryReceiver implements ITelemetryReceiver {
       listId: ENDPOINT_TRUSTED_APPS_LIST_ID,
       page: 1,
       perPage: 10_000,
-      filter: undefined,
+      filter: `exception-list-agnostic.attributes.created_at > ${moment
+        .utc()
+        .subtract(24, 'hours')
```

Comment:
You are also going to have introduce this filter to the following functions 

  - `fetchEndpointList`
  - `fetchDetectionExceptionList`